en/docs/quick-start-guides/deploy-your-first-integration-as-api.md

```diff
@@ -0,0 +1,198 @@
+# Deploy Your First Integration as API
+
+## Overview
+
+In this guide, you will:
+
+- Create a simple Integration as an API that calls a backend service and returns its response to the client. 
+- The backend service responds with a JSON with a key `message` and the value `Hello World!!!`, and the API deployed in Devant forwards this response to the client.
+- Use WSO2 Micro Integrator (MI) to develop the API.
+- Push the integration to Devant, which automatically builds and deploys the integration to the development environment.
+- Test the integration.
+
+## Prerequisites
+
+1. GitHub account: [Create a GitHub account](https://github.com/signup) if you don't have one already.
+2. If you're signing in to the Devant for the first time, create an organization:
+    1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in using your Google, GitHub, or Microsoft account.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use and Click **Create**.
+3. VSCode: [Install VSCode](https://code.visualstudio.com/download) if you don't have it installed already.
+
+## Step 1: Attach a Git repository
+1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in. This opens the organization overview page and lists the projects.
```

Comment:
let's use `https://console.devant.dev/new` url. This will land on the integration creation page without needing to select the project

Reply:
I don't think we should do that since selecting or creating a project is part of the flow. We should not hide it

Reply:
it's devant's design. we create a project by default for the user and land there. hence, this aligns with that approach